qiskit_nature/problems/second_quantization/molecular/molecular_problem.py

```diff
@@ -0,0 +1,111 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Molecular Problem class."""
+from typing import List, Optional, Tuple
+
+import numpy as np
+
+from qiskit_nature.drivers.qmolecule import QMolecule
+from qiskit_nature.drivers import FermionicDriver
+from qiskit_nature.operators import FermionicOp
+from qiskit_nature.operators.second_quantization import SecondQuantizedOp
+from qiskit_nature.problems.second_quantization.molecular.integrals_calculators \
+    .angular_momentum_integrals_calculator import \
+    calc_total_ang_momentum_ints
+from qiskit_nature.problems.second_quantization.molecular.fermionic_op_builder import \
+    build_fermionic_op, \
+    build_ferm_op_from_ints
+from qiskit_nature.problems.second_quantization.molecular.integrals_calculators \
+    .magnetization_integrals_calculator \
+    import calc_total_magnetization_ints
+from qiskit_nature.problems.second_quantization.molecular.integrals_calculators \
+    .particle_number_integrals_calculator \
+    import \
+    calc_total_particle_num_ints
+from qiskit_nature.transformations.second_quantization import BaseTransformer
+
+
+class MolecularProblem:
+    """Molecular Problem"""
+
+    def __init__(self, fermionic_driver: FermionicDriver,
+                 second_quantized_transformations: Optional[List[BaseTransformer]]):
```

Comment:
We need to adapt the naming of this variable as these are no longer linked to the `second_quantization` modules which we have scattered across Qiskit Nature.
For now, you can call them `q_molecule_transformers` or something alike.